pipeline/scripts/ci/upstream_cli.py

```diff
@@ -0,0 +1,160 @@
+"""
+This script provides the upstream latest rpm repo path and container image.
+Please note that this script should be run under root privilages.
+and podman should be installed on that host.
+"""
+import logging
+import os
+import re
+import sys
+
+import requests
+import yaml
+from docopt import docopt
+
+LOG = logging.getLogger(__name__)
+
+
+def exec_sys_commands(cmdlist):
+    """Method to execute system related commands.
+    Args:
+        cmdlist: List of commands to execute.
+    """
+    for cmd in cmdlist:
+        exit_status = os.system(cmd)
+        if exit_status:
+            raise Exception(f"Command failed to execute {cmd}")
+
+
+def store_in_upstream(branch_name, repo, image, version):
+    """Method to update it in upstream.yaml.
+    Args:
+        branch_name: Name of upstream branch
+        repo: upstream branch compose URL
+        image: upstream branch image
+        version: upstream branch ceph version.
+    """
+    url = "http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/upstream.yaml"
```

Comment:
As this script is understand the `pipeline`, we can safely assume that it works only in an CI environment. Hence, we can look at the mount directory for the specific file.

Reply:
Yes We can do that.

Reply:
You could read the file as the directory is mounted. Unless the intent is to execute the script outside of a pipeline environment.

Reply:
yes read the file from a mounted directory and removed that URL.